frontend/src/components/PatternflyComponents/Table/TableView.js

```diff
@@ -1,26 +1,88 @@
 
-import {Table, Thead, Tr, Th, Tbody, Td} from '@patternfly/react-table';
+import {Table, Thead, Tr, Th, Tbody, Td, ExpandableRowContent} from '@patternfly/react-table';
 import {Puff} from "react-loading-icons";
-import React from "react";
+import React, {useEffect, useState} from "react";
+import {BenchmarkResults} from "../../Home/BenchmarkResults";
+
+
+export const TableView = ({columns , rows = [], initialState = true, stickyHeader=false,
+                              addExpandableRows = false,  ...props  }) => {
+    /*
+        rows: {
+        dataset: {} // Complete data object
+        tableRows: {} // Table values that need to be displayed
+        }
+    */
```

Comment:
Hi @chentex 
I updated the table row format to get the data object to display the benchmark config results.